recipe/bld.bat

```diff
@@ -0,0 +1,42 @@
+:: cmd
+echo "Building %PKG_NAME%."
+
+
+:: Isolate the build.
+mkdir Build-%PKG_NAME%
+cd Build-%PKG_NAME%
+if errorlevel 1 exit /b 1
+
+
+:: Generate the build files.
+echo "Generating the build files..."
+cmake .. %CMAKE_ARGS% ^
+      -G"Ninja" ^
+      -DCMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^
+      -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+      -DCMAKE_BUILD_TYPE=Release
+
+
+:: Build.
+echo "Building..."
+ninja
+if errorlevel 1 exit /b 1
+
+
+:: Perform tests.
+::  echo "Testing..."
+::  ninja test
+::  path_to\test
+::  ctest -VV --output-on-failure
```

Comment:
Regression tests?

Reply:
Added the tests in a new commit.
The result is commented out because some tests are failing.